_docs/diverses/github_pr.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Github Pull-Request testen
+authors: [bloep]
+prio:
+---
+
+# Github Pull-Request testen
+
+- [Beschreibung](#beschreibung)
+- [Pull-Reuqest auschecken](#pr-auschecken)
+- [Fehler mitteilen](#fehler-mitteilen)
+	- [Codezeile kommentieren](#codezeile-kommentieren)
+	- [Anderungen gutheißen](#aenderungen-gutheissen)
+
+
+<a name="beschreibung"></a>
+## Beschreibung
+
+Nachfolgend möchte ich zeigen, wie einfach es ist, mit dem Tool [**Github hub**](https://hub.github.com/) Pull-Requests von Github in die lokale Entwicklungsversion von REDAXO zu laden, damit diese getestet werden können.
+
+
+**Für die nächsten Schritte wird vorrausgesetzt, dass [hub](https://hub.github.com/) installiert ist.**
+
+<a name="pr-auschecken"></a>
+## Pull Request auschecken
+
+In der Redaxo Entwicklungsversion folgenden Befehl im Terminal/Kommandozeile ausführen:
```

Comment:
Sollte man hier noch ergänzen, in welchem pfad man stehen sollte wenn man den checkout startet.